route/route.go

```diff
@@ -12,10 +12,15 @@ type Route struct {
 
 type controller interface {
 	Getter
+	Updater
 }
 
 type Getter interface {
-	GetSong(w http.ResponseWriter, r *http.Request)
+	GetUniversity(w http.ResponseWriter, r *http.Request)
+}
+
+type Updater interface {
```

Comment:
comments

Reply:
This should be more descriptive as if this is updater for controller? but good use of one method per interface